src/vs/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -233,16 +271,16 @@ const extensionPackStrings: Strings = {
 		]
 	}
 */
-const keymapStrings: Strings = {
-	installEvent: 'installKeymap',
-	installedEvent: 'installedKeymap',
-	detailsEvent: 'detailsKeymap',
-
-	alreadyInstalled: localize('welcomePage.keymapAlreadyInstalled', "The {0} keyboard shortcuts are already installed."),
-	reloadAfterInstall: localize('welcomePage.willReloadAfterInstallingKeymap', "The window will reload after installing the {0} keyboard shortcuts."),
-	installing: localize('welcomePage.installingKeymap', "Installing the {0} keyboard shortcuts..."),
-	extensionNotFound: localize('welcomePage.keymapNotFound', "The {0} keyboard shortcuts with id {1} could not be found."),
-};
+// const keymapStrings: Strings = {
```

Comment:
Probably should just delete if we don't want.  But again, I'd try to refactor the code such that we aren't making these edits under `vs` directory files.  This will cause unnecessary merge conflicts.